src/contents/api.ts

```diff
@@ -20,7 +27,16 @@ script.setAttribute("src", injectedScript);
 container.insertBefore(script, container.children[0]);
 container.removeChild(script);
 
-// receive API calls
+// Receive API calls:
```

Comment:
@7i7o See here.

Reply:
Ok. I have some thoughts on the iframe that I'll comment to you later on